lib/Horde/Mime/Headers/ContentParam.php

```diff
@@ -403,29 +403,46 @@ public function offsetUnset($offset)
     /**
      */
     public function serialize()
+    {
+        return serialize($this->__serialize());
+    }
+
+    /**
+     */
```

Comment:
okay i like it picasso

Reply:
it looks strange in the diff but it is the original docblock of ``unserialize`` :)